### ISSUE TITLE
cppcheck: out of bounds access

### ISSUE THREAD

**Issue by win32asm** [fukuchi/libqrencode#59]
ARGB has 4 components to save now...
```
diff --git a/qrenc.c b/qrenc.c
index 78f1c12..796ab01 100644
--- a/qrenc.c
+++ b/qrenc.c
@@ -189,7 +189,7 @@ static int color_set(unsigned char color[4], const char *value)
 {
        int len = strlen(value);
        int i, count;
-       unsigned int col[3];
+       unsigned int col[4];
        if(len == 6) {
                count = sscanf(value, "%02x%02x%02x%n", &col[0], &col[1], &col[2], &len);
                if(count < 3 || len != 6) {
```

**Comment by win32asm** [comment]
relevant pull request: https://github.com/fukuchi/libqrencode/pull/63

**Comment by fukuchi** [comment]
Thank you for the notification. Now I merged #63 and closed this bug.
